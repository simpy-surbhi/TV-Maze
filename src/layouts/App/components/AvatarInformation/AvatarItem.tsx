import { Typography, Avatar } from "@material-ui/core";
import * as React from "react";
import { useStyles } from "./styles";

interface Props {
  primaryText: string;
  secondaryText: string;
  image: string;
}

export const AvatarItem: React.FC<Props> = ({
  primaryText,
  secondaryText,
  image,
}) => {
  const classes = useStyles();
  return (
    <React.Fragment>
      <div className={classes.avatarItem}>
        <Avatar src={image} className={classes.large} />
        <div>
          <Typography variant="subtitle2">
            <strong>{primaryText}</strong>
          </Typography>
          <Typography variant="subtitle2">{secondaryText}</Typography>
        </div>
      </div>
    </React.Fragment>
  );
};
