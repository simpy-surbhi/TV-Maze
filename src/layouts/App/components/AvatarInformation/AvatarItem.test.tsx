import React from "react";
import { render } from "@testing-library/react";
import { AvatarItem } from "./AvatarItem";

test("AvatarItem renders correctly", () => {
  const { container } = render(
    <AvatarItem
      primaryText="Click Me"
      secondaryText="sd"
      image="https://static.tvmaze.com/uploads/images/original_untouched/11/27896.jpg"
    />
  );
  expect(container.firstChild).toMatchSnapshot();
});
