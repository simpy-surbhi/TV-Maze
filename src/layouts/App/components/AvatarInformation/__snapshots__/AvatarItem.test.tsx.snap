// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AvatarItem renders correctly 1`] = `
<div
  class="makeStyles-avatarItem-1"
>
  <div
    class="MuiAvatar-root MuiAvatar-circular makeStyles-large-2"
  >
    <img
      class="MuiAvatar-img"
      src="https://static.tvmaze.com/uploads/images/original_untouched/11/27896.jpg"
    />
  </div>
  <div>
    <h6
      class="MuiTypography-root MuiTypography-subtitle2"
    >
      <strong>
        Click Me
      </strong>
    </h6>
    <h6
      class="MuiTypography-root MuiTypography-subtitle2"
    >
      sd
    </h6>
  </div>
</div>
`;
